File: src/main/java/um/haberes/report/model/haberes/core/GeograficaDto.java
package um.haberes.report.model.haberes.core;

import java.math.BigDecimal;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import um.haberes.report.util.Jsonifyable;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class GeograficaDto implements Jsonifyable {

    private Integer geograficaId;

    private String nombre = "";

    private String reducido = "";

    private BigDecimal desarraigo = BigDecimal.ZERO;

    private Integer geograficaIdReemplazo;

}

File: src/main/java/um/haberes/report/kotlin/dto/haberes/core/GeograficaDto.kt
package um.haberes.report.kotlin.dto.haberes.core

import java.math.BigDecimal

data class GeograficaDto(

    var geograficaId: Int? = null,
    var nombre: String = "",
    var reducido: String = "",
    var desarraigo: BigDecimal = BigDecimal.ZERO,
    var geograficaIdReemplazo: Int? = null

)
